docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,113 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods.
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Cluster are limited to 2vCPU / 4GB memory. In addition, there are some pods other than ScalarDB Cluster pods on the worker nodes.
+
+In other words, the following components run on one worker node:
+
+* ScalarDB Cluster pod (2vCPU / 4GB)
+* Envoy proxy if you use `indirect` client mode or use other programming languages than Java (0.2–0.3 vCPU / 256–328 MB)
+* Kubernetes components
+* Your application pod
```

Comment:
```suggestion
* Your application pod (if you choose to run your application's pods in the same worker node)
```
I'm not sure, but something like the above might be needed to specify what we mean.

Reply:
I agree with you. We should make it clear. I will adjust the whole of this section based on the discussion.